recipes/toyplot/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "toyplot" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f3c07ed64609fa8747210421717b241c808da2fc5d296615c83bf422faca9e90
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - arrow
+    - custom_inherit
+    - multipledispatch
+    - numpy >=1.8.0
+    - pip
+    - pypng
+    - python
+    - reportlab
+    - six
+  run:
+    - arrow
+    - custom_inherit
+    - multipledispatch
+    - numpy >=1.8.0
+    - pypng
+    - python
+    - reportlab
+    - six
+
+test:
+  imports:
+    - toyplot
+    - toyplot.reportlab
+
+about:
+  home: "http://toyplot.readthedocs.org"
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
Add a comment here with the following link as a reminder to include the upstream license if it is added in a future release:

https://github.com/sandialabs/toyplot/issues/197